app/src/main/res/values/strings.xml

```diff
@@ -669,31 +669,41 @@
   <string name="backup_password">Password</string>
+  <string name="backup_body_title">Backup your wallet key now!</string>
   <string name="backup_body">We don\'t store your private keys or any other access method. It\'s important to keep your wallet\'s backup (keystore) safe.</string>
-  <string name="backup_confirmation_title">Just one more step</string>
-  <string name="backup_confirmation_body">Keep your backup safe! Where do you want to save it?</string>
-  <string name="backup_confirmation_yes">YES, FINISH</string>
-  <string name="backup_confirmation_no">NO, SAVE ON DEVICE</string>
+  <string name="backup_confirmation_title">Your backup is ready!\n<br />Keep it safe</string>
+  <string name="backup_option_email_title">Save on your email</string>
+  <string name="backup_options">Or</string>
+  <string name="backup_option_email_hint">Your email here…</string>
+  <string name="backup_option_email_button">Send to email</string>
+  <string name="backup_option_device_button">Save on device</string>
   <string name="backup_done_title">Backup done!</string>
   <string name="backup_done_body">Are you sure your backup file is on a safe place?</string>
   <string name="backup_confirmation_no_share_title">Backup done!</string>
-  <string name="backup_confirmation_no_share_body">Your backup file is in your Downloads folder.</string>
-  <string name="backup_save_dialogue_title">Save backup file</string>
-  <string name="backup_save_dialogue_body">Choose a name for your backup file.</string>
-  <string name="backup_confirmation_tips_title">Keep the file safe:</string>
-  <string name="backup_confirmation_tips_1">Print it and store it somewhere safe.</string>
-  <string name="backup_confirmation_tips_2">Save it on a USB drive.</string>
-  <string name="backup_confirmation_tips_3">Copy it to a private and secure cloud storage.</string>
+  <string name="backup_confirmation_no_share_body">Your backup file is stored in your device.</string>
+  <string name="backup_confirmation_button">I understand</string>
+  <string name="backup_confirmation_email_body">Your backup file is in your email.</string>
+  <string name="backup_save_dialogue_title">Save Backup</string>
+  <string name="backup_save_dialogue_body">Please enter a name for the backup file.</string>
+  <string name="backup_confirmation_tips_title">Keep the file safe!</string>
+  <string name="backup_confirmation_tips_1_title">Print it</string>
+  <string name="backup_confirmation_tips_1_body"> and store it somewhere safe.</string>
+  <string name="backup_confirmation_tips_2_title">Save it</string>
+  <string name="backup_confirmation_tips_2_body"> on a USB drive.</string>
+  <string name="backup_confirmation_tips_3_title">Copy it</string>
+  <string name="backup_confirmation_tips_3_body"> to a private and secure cloud storage.</string>
+  <string name="backup_skip_title">I understand that I may loose my funds if I don\'t backup my wallet key</string>
```

Comment:
this is though because simply changing it, would create inconsistencies with other languages.
We need to see this with Carlos, because new strings we can simply add, and then the translation will be done, but replacing old strings might be a problem.
To be honest, this is not being well managed and this problem should never be a problem. In short, before we develop the feature, this file should already be updated.
Because its not being well managed, doing this change might cause the problem to fall over us

Reply:
i had previously discussed this with aleixo who told me to do it this way and that carlos would be able to see the differences, and to only concern myself with doing the changes in this file.. but i'll check with carlos